src/site/content/en/blog/samesite-cookie-recipes/index.md

```diff
@@ -0,0 +1,324 @@
+---
+title: SameSite cookie recipes
+subhead:
+  Update your site's cookies to prepare for the upcoming changes to the SameSite
+  attribute's behavior.
+authors:
+  - rowan_m
+date: 2019-10-15
+hero: cookie-hero.jpg
+description: |
+  With the introduction of the new SameSite=None attribute value, sites can now
+  explicitly mark their cookies for cross-site usage. Browsers are moving to
+  make cookies without a SameSite attribute act as first-party by default, a
+  safer and more privacy preserving option than the current open behavior.
+  Learn how to mark up your cookies to ensure your first-party and third-party
+  cookies continue to work once this change comes into effect.
+tags:
+  - post
+  - security
+  - cookies
+---
+
+{% Aside %}
+
+If you need a refresher on cookies and the `SameSite` attribute, head over to
+the earlier companion article,
+["SameSite cookies explained"](/samesite-cookies-explained).
+
+{% endAside %}
+
+To recap from before: [Chrome](https://www.chromium.org/updates/same-site),
+[Firefox](https://groups.google.com/d/msg/mozilla.dev.platform/nx2uP0CzA9k/BNVPWDHsAQAJ),
+[Edge](https://textslashplain.com/2019/09/30/same-site-cookies-by-default/), and
+others will be changing their default behavior so that:
+
+1. cookies without a `SameSite` attribute will be treated as `SameSite=Lax`, or
+   restricted to first-party contexts
```

Comment:
@robdodson @kaycebasques This is a case where our practice of presenting everything as shiny may not be serving the internet well. There needs to be some attention to this at least in the description, because this is going to break the web.

Reply:
I didn't at first realize there were two articles here. This comment applies to the announcement article but not this one.

Reply:
@jpmedley I updated the title of this PR to clarify that there are 2 posts. Regarding your first comment, can you provide suggestions on what needs to change and what we should change it to?

Reply:
Maybe "Same site cookies are coming. Make sure you're ready?"

I don't know how to be strong enough without being negative. We have to say, "We're about to break your site", without actually saying it.

Reply:
I've tried to make this more explicit in the latest edits.